src/test/pages/Burn.test.tsx

```diff
@@ -12,7 +12,7 @@ describe('Burn page', () => {
   it('should display burn tab when there is liquidated vault', async () => {
     await render(<App />, { path: '/bridge?tab=burn' });
 
-    const burnTab = await screen.findByText(/Burn/i);
+    const burnTab = screen.getByRole('tab', { name: /burn/i });
```

Comment:
@peterslany 
I made this change as somewhere I read that `getByRole` should be used as much as possible. Please correct me if I'm wrong. Thank you.